stdlib/public/core/StringGuts.swift

```diff
@@ -51,10 +51,22 @@ extension _StringGuts {
   }
 
   @inlinable @inline(__always)
-  internal init(_ bufPtr: UnsafeBufferPointer<UInt8>, isASCII: Bool) {
+  internal init(immortal bufPtr: UnsafeBufferPointer<UInt8>, isASCII: Bool) {
     self.init(_StringObject(immortal: bufPtr, isASCII: isASCII))
   }
 
+  @usableFromInline
```

Comment:
Should this be:

```suggestion
  @usableFromInline @inline(__always)
```

I couldn't find any `@inline(__always)` documentation, and I don't know if this [overview](https://forums.swift.org/t/inline-best-practices/5353/3) is up to date.